scripts/shapella-upgrade/deploy-shapella-contracts.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+set -e +u
+set -o pipefail
+
+ARAGON_APPS_REPO_REF=import-shared-minime
+
+if [[ -z "${DEPLOYER}" ]]; then
+  echo "Env variable DEPLOYER must be set"
+  exit 1
+fi
+echo "DEPLOYER is $DEPLOYER"
+
+if [[ -z "${NETWORK}" ]]; then
+  echo "Env variable NETWORK must be set"
+  exit 1
+fi
+echo "NETWORK is $NETWORK"
+
+# yarn install --immutable
+# yarn compile
+
+rm -f deployed-$NETWORK.json
+cp deployed-$NETWORK-defaults.json deployed-$NETWORK.json
+
+yarn hardhat --network $NETWORK run ./scripts/shapella-upgrade/deploy-aragon-implementations.js
+
```

Comment:
it's not clear who and when deploys non-aragon impls

Reply:
these instructions belong to the deploy plan, don't they?